cmd/livepeer/starter/starter.go

```diff
@@ -5,8 +5,8 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"github.com/ethereum/go-ethereum/common/math"
 	"io/ioutil"
+	"math"
```

Comment:
This is an intentional correction. Math should be from regular class, not ethereum package.

https://github.com/livepeer/go-livepeer/blob/484fbe4091af9666b344ce2b824393099237de57/cmd/livepeer/starter/starter.go#L158